test/services/tagged_organisations_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
+require './test/support/custom_assertions.rb'
+
+describe TaggedOrganisations do
+  describe '#fetch' do
+    it 'return tagged organisations' do
+      search_results = {
+        'results' => [],
+        'aggregates' => {
+          'organisations' => {
+            'options' => [{
+              'value' => {
+                'title' => 'Department for Education',
+                'content_id' => 'ebd15ade-73b2-4eaf-b1c3-43034a42eb37',
+                'link' => '/government/organisations/department-for-education',
+                'slug' => 'department-for-education',
+                'organisation_state' => 'live'
+              },
+              'documents' => 89
+            }]
+          }
+        }
+      }
+
+      Services.rummager.stubs(:search).returns(search_results)
+
+      organisations = tagged_organisations.fetch
+      assert_equal(1, organisations.count)
+      assert_equal('Department for Education', organisations.first.title)
+      assert_equal(89, organisations.first.document_count)
+    end
+
+    it 'only returns live organisations' do
+      search_results = {
```

Comment:
Similarly, these tests span many lines due to the dependency setup, consider using `before` and `let` to break them up into setup and assertion.

Reply:
Done https://github.com/alphagov/collections/pull/586/files#diff-62314a7230a7329a26b5cb721a328944